examples/example_minimal.py

```diff
@@ -41,9 +41,8 @@ def f_target(x):
 # on a set of random points.
 
 
+@cgp.utils.objective
```

Comment:
as we'll be using this in every script, maybe pull it up into the cgp namespace by importing it in `cgp/__init__.py`? that would allows us to write `@cgp.objective` which seems nicer.